provision-contest/ansible/roles/ssh/tasks/main.yml

```diff
@@ -39,3 +39,10 @@
   loop:
     - domjudge
     - root
+
+- name: Ensure authorized_keys for GitHub user accounts are present.
+  authorized_key:
+    user: "{{ ssh_user.user }}"
```

Comment:
Where is this coming from?

Reply:
Wrong cherry-picking, in my next PR I swap 2 files around and then this will make sense. I'll fix it here.